### ISSUE TITLE
If field label is hidden, it should output as screen reader text

### ISSUE THREAD

**Issue by Shelob9** [CalderaWP/Caldera-Forms#1314]
No label is 10000% unacceptable for accessibility reasons
Related #1313

**Comment by Shelob9** [comment]
Close via 	6f9186e
